src/test/java/com/kodilla/ecommercee/ProductTestSuite.java

```diff
@@ -0,0 +1,55 @@
+package com.kodilla.ecommercee;
+
+import com.kodilla.ecommercee.dao.ProductDao;
+import com.kodilla.ecommercee.domain.Product;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import static org.junit.Assert.assertEquals;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class ProductTestSuite {
```

Comment:
tak klasa nie powinna być w tasku JDP190401-17?

Reply:
Generalnie implementując jakąś funkcjonalność, chcąc zamknąć brancha trzeba dostarczyć również testy jednostkowe. W jirze jest to rozbite na 2 osobne taski więc zaproponowałem kompromis. W tym PR dodamy podstawowy test sprawdzający czy klasy Product i ProductDao zapisują się w bazie. W  tasku JDP190401-17 zrobimy szczegółowe testy z relacjami itp